src/applications/vaos/new-appointment/components/TypeOfCarePage/index.jsx

```diff
@@ -83,18 +86,19 @@ export default function TypeOfCarePage() {
   return (
     <div>
       <h1 className="vads-u-font-size--h2">{pageTitle}</h1>
-      {showUpdateAddressAlert && (
-        <UpdateAddressAlert
-          onClickUpdateAddress={heading => {
-            dispatch(clickUpdateAddressButton());
-            recordEvent({
-              event: 'nav-warning-alert-box-content-link-click',
-              alertBoxHeading: heading,
-            });
-            resetDataLayer();
-          }}
-        />
-      )}
+      {showUpdateAddressAlert &&
+        !featureCCIteration && (
```

Comment:
I don't see this change mentioned in the ticket, we might want to keep it, because not having an address impacts the community eligibility check. Could check with @laurenernest.

Reply:
Agreed. No changes to the update address alert or type of care page in this ticket.